docs/src/components/ColorSwatch/ColorSwatch.js

```diff
@@ -38,6 +38,9 @@ const ColorSwatch = props => {
   if (props.border) {
     classNames.push(getClassName('bpkdocs-color-swatch--border'));
   }
+  if (props.className) {
+    classNames.push(props.className);
+  }
 
```

Comment:
Can we potentially change this code to be more slick by doing something like this?

const classNames = getClassName(
    'bpkdocs-color-swatch',
    props.whiteColor && 'bpkdocs-color-swatch--light',
    props.border && 'bpkdocs-color-swatch--border',
    props.className,
  );

Reply:
Done